src/OrchardCore/OrchardCore.ContentManagement/Handlers/ContentPartHandlerCoordinator.cs

```diff
@@ -603,13 +290,15 @@ public override async Task CloningAsync(CloneContentContext context)
             foreach (var typePartDefinition in contentTypeDefinition.Parts)
             {
                 var partName = typePartDefinition.PartDefinition.Name;
+                if (String.IsNullOrEmpty(partName))
```

Comment:
Use curly bracket. Log the error and continue execution instead of throwing an exception.